app/assets/javascripts/single_page/dynamic_table.js.erb

```diff
@@ -582,16 +610,16 @@ function inputObjectsInput(column, data, options, url){
 }
 
 function cvListObjectsInput(column, data, options, url){
-  const existingOptions = data.map((e) => `<option selected="selected" value="${e}">${e}</option>`);
-  if (options.readonly) {
-    return data.map((e) => `<span title="ID: ${e}" class="badge">${e}</span>`).join(" ");
-  } else {
-    const typeaheadTemplate = 'typeahead/controlled_vocab_term';
-    const objectInputName = data.map((e) => e.title).join('-') + '-' + crypto.randomUUID();
-    const extraClass = '';
-    const titleText = '';
-    const allowNewItems = column.cv_allows_free_text;
-    setTimeout(ObjectsInput.init);
+  const existingOptions = data.map((e) => `<option selected="selected" value="${sanitizeData(e)}">${sanitizeData(e)}</option>`);
+    if (options.readonly) {
+        return data.map((e) => `<span title="ID: ${sanitizeData(e)}" class="badge">${sanitizeData(e)}</span>`).join(" ");
+    } else {
+        const typeaheadTemplate = 'typeahead/controlled_vocab_term';
+        const objectInputName = data.map((e) => sanitizeData(e)).join('-') + '-' + crypto.randomUUID();
+        const extraClass = '';
```

Comment:
could `sanitizeData(e)` be called once at the start and assigned to a variable, instead of being called repeatedly?

Reply:
looking at it again, I can see that e may be different in each case, so ignore this